IPython/html/base/handlers.py

```diff
@@ -250,6 +255,7 @@ def template_namespace(self):
             sys_info=sys_info,
             contents_js_source=self.contents_js_source,
             version_hash=self.version_hash,
+            **self.jinja_template_vars
```

Comment:
Sorry, I missed your first note. Looking at the list here, I could imagine wanting to override the various URLs, contents_js_source, and version_hash, but I'm inclined to say that I'd want to override them explicitly as separate configurables rather than all at once with this mechanism.  Are any of those attributes not yet configurable themselves via alternative means?